src/job_application_profile.py

```diff
@@ -5,6 +5,21 @@
 from loguru import logger
 
 
+@dataclass
+class PersonalInformation:
+    name: str
+    surname: str
+    date_of_birth: str
+    country: str
+    city: str
+    address: str
+    phone_prefix: str
+    phone: str
+    email: str
+    github: str
+    linkedin: str
```

Comment:
shouldn't this field be optional?
or have default value of NA
- github can be NA
- linkedin can be NA as some users may not be interested to provide